frontend/lib/package.json

```diff
@@ -0,0 +1,151 @@
+{
+    "name": "@streamlit/lib",
+    "version": "1.21.0",
```

Comment:
(See previous comment re: auto-updating this version string in our `update_version.py` script)